README.md

```diff
@@ -82,13 +111,19 @@ The root module serves as the entry point for one-app to load an application.
           | ------------------------------- |
 ```
 
-It is possible for your application to consist of only the root module, however most application will want to take advantage of code splitting using [Holocron](https://github.com/americanexpress/holocron) and have the root module load other modules.
+It is possible for your application to consist of only the root module, however most application will want to take advantage of code splitting using [Holocron](https://github.com/americanexpress/holocron) and have the root module load other modules. More on this in the [Routing](#-routing) section in the API docs.
 
 For a module to act as the root module the only requirements are:
 
 - Returns a React component bundled with [one-app-bundler](https://github.com/americanexpress/one-app-cli).
 - Provides a valid [content security policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) though the [appConfig](#-configuration-with-app-config) static.
 
+**📘 More Information**
+* Root Module example: [frank-lloyd-root](https://github.com/americanexpress/one-app/blob/master/prod-sample/sample-modules/frank-lloyd-root/0.0.0/src/components/FrankLloydRoot.jsx)
+* [App Configuration in your Root Module](#-app-configuration)
+* [What are Holocron Modules?](#-modules)
+* [Useful Local Development Commands / Options](#-useful-local-development-commands-options)
```

Comment:
I think you need to remove the `-` from the `#` links as it doesn't seem to work. https://github.com/americanexpress/one-app/blob/5d97705b1c6e7a2331b16c6dd4dcc7d77580c185/README.md#-app-configuration

Reply:
good catch! Combed thru the doc and turns out a few more links in addition to those were broken. Anchor links are finnicky. fixed in 42f589c

Reply:
Yes they are. 👍